test/e2e/traceflow_test.go

```diff
@@ -337,10 +337,6 @@ func testTraceflowIntraNode(t *testing.T, data *TestData) {
 	}
 
 	expectedTTL := int32(64)
-	if testOptions.enableAntreaIPAM {
-		// AntreaIPAM will always overwrite dstMAC and decrease TTL
-		expectedTTL = 63
-	}
```

Comment:
Do we have doc about `ModDstMAC` and `DecTTL` behavior?

Reply:
No, I plan to doc about flexible pipeline in another PR.